pkg/virt-api/webhooks/defaults.go

```diff
@@ -50,9 +50,30 @@ func SetDefaultVirtualMachineInstance(clusterConfig *virtconfig.ClusterConfig, v
 	setDefaultHypervFeatureDependencies(&vmi.Spec)
 	setDefaultCPUArch(clusterConfig, &vmi.Spec)
 	setGuestMemoryStatus(vmi)
+	setCurrentCPUTopologyStatus(vmi)
 	return nil
 }
 
+func setCurrentCPUTopologyStatus(vmi *v1.VirtualMachineInstance) {
```

Comment:
This function is very different from the one it replaces...
Shouldn't we use the values from `SetDefaultGuestCPUTopology()` instead of all `1`s?
It also clears `vmi.Status.CurrentCPUTopology` when `vmi.Spec.Domain.CPU` is non-nil, why?

Reply:
> Shouldn't we use the values from SetDefaultGuestCPUTopology() instead of all 1s?

Actually no, that's already called by `setDefaultVirtualMachineInstanceSpec`  so we should always have `vmi.Spec.Domain.CPU` populated now so I think we can remove L58-62 entirely. I'll respin and confirm with a unit test.